src/main/java/org/geysermc/discordbot/tags/TagsManager.java

```diff
@@ -136,23 +137,24 @@ private static void loadTags() {
                                     if (tagData.containsKey("aliases")) {
                                         GeyserBot.LOGGER.warn("Tag '" + tagName + "' has aliases listed but is of type 'issue-only'. Ignoring aliases.");
                                     }
+
                                     if (tagData.containsKey("image")) {
                                         GeyserBot.LOGGER.warn("Tag '" + tagName + "' has image listed but is of type 'issue-only'. Ignoring image.");
                                     }
+
                                     if (issueTriggers == null) {
                                         GeyserBot.LOGGER.warn("Tag '" + tagName + "' has no issues listed but is of type 'issue-only'. Ignoring tag.");
-                                        continue;
+                                        break;
+                                    }
+
+                                    for (String issue : issueTriggers) {
+                                        ISSUE_RESPONSES.put(issue, content.toString());
                                     }
+                                    break;
 
                                 default:
                                     GeyserBot.LOGGER.warn("Invalid tag type '" + tagData.get("type") + "' for tag '" + tagName + "'!");
-                                    continue;
-                            }
-
-                            if (issueTriggers != null) {
-                                for (String issue : issueTriggers) {
-                                    ISSUE_RESPONSES.put(issue, content.toString());
-                                }
```

Comment:
It would be useful to let tags other than type `issue-response` have `issues` listed

With these changes, if we wanted to be able to call a tag through the tag command and also have it as an issue response, we would have to have two different tag files with identical contents

Reply:
Ah feel free to revert it, wasn't trying to have any regressions to the last commit just cleanup mainly.